.github/workflows/deploy.yml

```
@@ -0,0 +1,177 @@
+        working-directory: ./frontend/web
+        run: npm run build
+        env:
+          REACT_APP_API_URL: https://us-central1-${{ secrets.GOOGLE_CLOUD_PROJECT }}.cloudfunctions.net/uruguay-news-api
+          REACT_APP_ENVIRONMENT: production
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v4
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./frontend/web/build
+          cname: uruguay-news.your-domain.com 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)